azure_media_services/ams.py

```diff
@@ -115,16 +121,42 @@ class AMSXBlock(StudioEditableXBlockMixin, XBlock):
         default=True,
         scope=Scope.settings
     )
+    assets_download = String(
+        display_name=_("Assets download mode"),
+        help=_("Disables completely or enables download controls in xBlock/Player."),
+        default=AssetsMode.edx,
+        values=[
+            {"display_name": "Edx - way (footer links)", "value": AssetsMode.edx},
```

Comment:
These values should be reviewed by the UX team.

Reply:
We will need to demo then, we fix it later.